presto-spi/src/main/java/com/facebook/presto/spi/SchemaTablePrefix.java

```diff
@@ -66,6 +66,14 @@ public boolean matches(SchemaTableName schemaTableName)
         return tableName == null || tableName.equals(schemaTableName.getTableName());
     }
 
+    public SchemaTableName toSchemaTableName()
```

Comment:
I like that! Search & destroy `new SchemaTableName(prefix.getSchemaName(), prefix.getTableName())` now :)